content/python/concepts/dictionaries/dictionaries.md

```diff
@@ -12,11 +12,11 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-A dictionary is an ordered set of key-value pairs. It provides a way to map pieces of data to each other and allows for quick access to values associated with keys. Keys must be unique in dictionaries, but the values don't have to be.
+A dictionary is a set of key-value pairs. It provides a way to map pieces of data to each other and allows for quick access to values associated with keys. Keys must be unique in dictionaries, but the values don't have to be.
```

Comment:
```suggestion
A dictionary is a data set of key-value pairs. It provides a way to map pieces of data to each other and allows for quick access to values associated with keys.
```

I don't believe the last sentence is needed because it is already mentioned on line # 35 in the "Syntax" section, where I think it might be a better fit.